### PR TITLE
#1813-Last-activity-link-do-not-work-anymore

### PR DESCRIPTION
**Pull request by christophscheunemann**
- Fix for issue #1813

### CODE REVIEW

File: inception-app-webapp/src/main/resources/application.properties [hunk 1]
```diff
@@ -66,3 +66,5 @@ wicket.stuff.htmlcompressor.features.removeIntertagSpaces=true
 wicket.stuff.htmlcompressor.features.preserveLineBreaks=true
 wicket.stuff.htmlcompressor.features.compressJavaScript=false
 wicket.stuff.htmlcompressor.features.compressCss=false
+
+workload.dynamic.enabled=true
```
Comment (jcklie) [inline inception-app-webapp/src/main/resources/application.properties:70]: No

Comment (christophscheunemann) [reply inception-app-webapp/src/main/resources/application.properties]: yes just seen and fixed

Comment (jcklie) [reply inception-app-webapp/src/main/resources/application.properties]: Thanks!

**Review by UWinch — APPROVED** [review]
Works again, thank you.

**Pull request merged**